apps/docs/docs/guides/platform/logs.mdx

```diff
@@ -110,6 +85,46 @@ The Logs tab displays logs emitted during function execution.
 </TabPanel>
 </Tabs>
 
+---
+
+## Advanced Configuration
+
+### Logging Postgres Queries
+
+By default, query logs are disabled for new Supabase projects, as they can reveal metadata about the contents of your database (such as table and column names).
+
+To enable query logs, enable and configure the [pgAudit extension](https://www.pgaudit.org/):
+
+1. [Enable the pgAudit extension](https://app.supabase.com/project/_/database/extensions)
+2. [Perform a "Fast database reboot"](https://app.supabase.com/project/_/settings/general)
```

Comment:
This option might be easy to miss since its behind a dropdown. Maybe we can say something like this instead?
```suggestion
2. [Restart your project](https://app.supabase.com/project/_/settings/general) using the **Fast database reboot** option.
```